makefile

```diff
@@ -77,4 +77,3 @@ server: $(patsubst $(SRC)/server/$(SRCOBJS), $(OBJS), $(wildcard $(SRC)/server/*
 # Deletes the executable and all .o and .d files in the bin folder
 clean: | $(ODIR)
 	$(RM) $(EXEC) $(wildcard $(ODIR)/server*) $(wildcard $(EXEC).*) $(wildcard $(ODIR)/*.d*) $(wildcard $(ODIR)/*.o)
-
```

Comment:
the makefile should not be changed in commits without direct instruction to do so, it at this point is rather complex and should not need further changes